recipe2023/index.html

```diff
@@ -1,80 +1,116 @@
+                <li>
+                  Serve in bowls or cups garnished with chopped green onions or
+                  chives. Remember to treat the wontons gently to prevent them
+                  from tearing! If you’d like to add veggies and shrimp to turn
+                  this into a Wor Wonton Soup, these wontons work well for that
+                  too!
+                </li>
+              </ol>
+            </div>
+          </div>
+          <div class="last-section">
+            <h3>Want Something Different?</h3>
 
-  <div>
-    <h3>Ingredients</h3>
-    <img
-      src="https://media.istockphoto.com/id/1372510923/photo/kulit-pangsit-or-dumpling-skin-usually-as-a-wrapper-with-minced-meat-in-it.jpg?s=612x612&w=0&k=20&c=h7CdGllDW19jdXwVaRPnNXyYWPYmOvNo_XM73tqtG3M="
-      alt="wonton wrapper"
-      width="600"
-      height="450"
-    />
-    <ul>
-      <li>8-10 Momos , only filled and shaped, not steamed</li>
-      <li>1 clove Garlic , finely chopped</li>
-      <li>1 inch Ginger , grated</li>
-      <li>1 or 2 Green Chillies , finely chopped</li>
-      <li>2 to 3 Spring Onion (Bulb & Greens) , finely chopped with leaves</li>
-      <li>1 teaspoon Light soy sauce</li>
-      <li>1/2 teaspoon Sesame (Gingelly) Oil</li>
-      <li>1 teaspoon Rice Vinegar</li>
-    </ul>
-  </div>
+            <h3>Don't worry, we got you.</h3>
 
-  <div>
-    <h3>How to make Wonton Soup</h3>
-    <ul>
-      <li>1.Place broth and seasoning in a pot and bring to a boil.</li>
-      <li>2.Remove the ginger and garlic and reduce heat.</li>
-      <li>3.Gently simmer the wontons in the broth until done.</li>
-      <li>
-        Serve in bowls or cups garnished with chopped green onions or chives.
-        Remember to treat the wontons gently to prevent them from tearing! If
-        you’d like to add veggies and shrimp to turn this into a Wor Wonton
-        Soup, these wontons work well for that too!
-      </li>
-    </ul>
-  </div>
+            <h3>Check Out These Links For Other Wonton Recipes:</h3>
 
-  <div>
-    <h3>Want Different Style Wontons?</h3>
-    <h4>
-      Don't worry, we got you.<br />
-      Check out these links for different recipes:
-    </h4>
-    <a href="https://www.spendwithpennies.com/homemade-wonton-soup/"
-      >Homemade Wonton Soup</a
-    >
-  </div>
-</body>
+            <a
+              class="link"
+              href="https://www.spendwithpennies.com/homemade-wonton-soup/"
+              ><button class="button">Homemade Wonton Soup</button></a
+            >
+
+            <a class="link" href="https://www.skinnytaste.com/wonton-soup/"
+              ><button class="button">SkinnyTaste</button></a
+            >
+            <a class="link" href="https://www.recipetineats.com/wonton-soup/"
+              ><button class="button">RecipeTin Eats</button></a
+            >
+            <br />
+          </div>
+        </div>
+      </div>
+    </div>
+    <script type="module" src="index.js"></script>
```

Comment:
Remove the `type="module"` from here, it's gonna cause problems running via the `file://` protocol and you don't need to make use of that functionality.